pages/index.js

```diff
@@ -56,6 +56,15 @@ export default function Home({
     }));
   };
 
+  const clearSituation = () => {
+    fetch(`${process.env.NEXT_PUBLIC_BASE_URL}/api/situation`, {
+      method: "delete",
+    });
```

Comment:
The cookie gets recreated immediately after by setting situation so this call doesn't really do anything